english/explore-outside/talk-child/smile/smile-to-child.md

```diff
@@ -0,0 +1 @@
+Even though the glaring red eyes are creepy, I decide to communicate with body languages. So I smile at the child reassuringly, extending my right hand towards him.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.